gatsby-config.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Configure your Gatsby site with this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/gatsby-config/
+ */
+
+module.exports = {
+  siteMetadata: {
+    title: `Will's Blog`,
```

Comment:
Just a style note — a pretty standard rule for quotes is:

- If it's a template string like `/${dirname}/src`, use `
- For anything else, use either " or ' (but always use the same one)

Installing [eslint](https://eslint.org/) and setting up a common ruleset like [eslint:recommended](https://eslint.org/docs/2.0.0/rules/) and setting it up in your code editor makes it easy to spot inconsistencies when coding. You typically set up default rules, then add some of your own overrides based on preferences. Depending on how much time you have this week, playing around with eslint on this project might be a good idea.

However..... @solomonhawk it's interesting to note that Gatsby docs use backticks for pretty much _every_ string: https://www.gatsbyjs.org/docs/gatsby-config/ — I hadn't seen this style before, but I kinda dig it? There's no performance overhead or anything, you just lose the visual cue that one string has a var in it, another doesn't. Thoughts?

Reply:
(Also good when setting up eslint: Add the [eslint-react-plugin](https://github.com/yannickcr/eslint-plugin-react), which can enforce style rules in your JSX)